README.md

```diff
@@ -19,44 +19,80 @@ We assume that you already have:
 
 Then, you can create an URS for your circuit in the following way:
 
-```rust,ignore
```

Comment:
I guess the description above the example could be updated

Reply:
Updated. Please have a check. 
Note I still use the name `circuit-construction` and the name `kimchi` still remains. I guess they can remain until a new name for `circuit-construction` is determined.